NullPointerException from QC page

I sometimes see this error in the QuinCe logs. Figure it out.

```
Sep 21 14:03:15 quince3 catalina.sh[235]: Caused by: java.lang.NullPointerException: Cannot invoke "uk.ac.exeter.QuinCe.data.Dataset.SensorValue.isNaN()" because "sensorValue" is null
Sep 21 14:03:15 quince3 catalina.sh[235]: #011at uk.ac.exeter.QuinCe.web.datasets.plotPage.ManualQC.ManualQCData.canSelectCell(ManualQCData.java:741)
```

--- a/WebApp/src/uk/ac/exeter/QuinCe/web/datasets/plotPage/ManualQC/ManualQCData.java
+++ b/WebApp/src/uk/ac/exeter/QuinCe/web/datasets/plotPage/ManualQC/ManualQCData.java
@@ -738,7 +738,7 @@ public class ManualQCData extends PlotPageData {
     if (selectable) {
       SensorValue sensorValue = sensorValues.get(DateTimeUtils.longToDate(row))
         .get(column);
-      if (sensorValue.isNaN() || isGhost(sensorValue)) {
+      if (null == sensorValue || sensorValue.isNaN() || isGhost(sensorValue)) {
         selectable = false;
       }
     }

